recipes/optional-lite/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,28 @@
+#include "nonstd/optional.hpp"
+
+#include <cstdlib>
+#include <iostream>
+
+using nonstd::optional;
+using nonstd::nullopt;
+
+optional<int> to_int( char const * const text )
+{
+    char * pos = NULL;
+    const int value = static_cast<int>( strtol( text, &pos, 0 ) );
+
+    return pos == text ? nullopt : optional<int>( value );
+}
+
+int main( int argc, char * argv[] )
+{
+    const char * text = argc > 1 ? argv[1] : "42";
+
+    optional<int> oi = to_int( text );
+
+    if ( oi ) std::cout << "'" << text << "' is " << *oi << std::endl;
+    else      std::cout << "'" << text << "' isn't a number" << std::endl;
+}
+
+// cl -nologo -W3 -EHsc -I../include to_int.cpp && to_int x1
+// g++ -Wall -Wextra -std=c++03 -I../include -o to_int.exe to_int.cpp && to_int x1
```

Comment:
What are these last two lines about?

(This pull request got my attention again as I realized that we have such a recipe in Bincrafters as well 😄)

Reply:
I copy/pasted the full example file from Martin's repo: https://github.com/martinmoene/optional-lite/blob/v3.2.0/example/01-to_int.cpp

Probably it is better to add a link